arcgis-ios-sdk-samples/Content Display Logic/Controllers/Color Picker/ColorPickerViewController.swift

```diff
@@ -54,10 +56,12 @@ class ColorPickerViewController: UITableViewController {
         hueSlider.value = Float(hue)
         saturationSlider.value = Float(saturation)
         brightnessSlider.value = Float(brightness)
+        alphaSlider.value = Float(alpha)
```

Comment:
It doesn't make a whole lot of sense to update every slider and label when a single component changes. Could you break this method up into a separate method per component?

Reply:
Yeah, you're right. As you can guess, this was faster to write 😆